save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -128,20 +130,32 @@ class FixPlugin(
                 }
             }
 
-    private fun createTestFile(path: Path, generalConfig: GeneralConfig): Path {
+    private fun createTestFile(
+        path: Path,
+        generalConfig: GeneralConfig,
+        fixPluginConfig: FixPluginConfig): Path {
         val pathCopy: Path = constructPathForCopyOfTestFile(FixPlugin::class.simpleName!!, path)
         createTempDir(pathCopy.parent!!)
 
-        val expectedWarningPattern = generalConfig.expectedWarningsPattern
+        val defaultIgnoreLinesPatterns: MutableList<Regex> = mutableListOf()
+        generalConfig.expectedWarningsPattern?.let { defaultIgnoreLinesPatterns.add(it) }
+        generalConfig.runConfigPattern?.let { defaultIgnoreLinesPatterns.add(it) }
 
         fs.write(fs.createFile(pathCopy)) {
-            fs.readLines(path).forEach {
-                if (expectedWarningPattern == null || !generalConfig.expectedWarningsPattern!!.matches(it)) {
+            fs.readLines(path)
+                .filter { line ->
+                    fixPluginConfig.ignoreLines?.let {
```

Comment:
what will happen here if `ignoreLines == null`? Sorry, if I am not completely understand this logic

Reply:
I have two variables: `ignoreLines` and `ignoreLinesPatterns`. We deserialize `ignoreLines` from toml config so it is nullable. Than we use `ignoreLinesPatterns` is a mutable list of Regex and it is not nullable.
In fix plugin there are three cases: `ignoreLines` is empty, `ignoreLines` is null or `ignoreLines` is a list of regular expressions. Here I check if `ignoreLines` is null